lib/src/enums.dart

```diff
@@ -41,6 +41,7 @@ enum Status {
 }
 
 enum CommentStatus {
+  approve,
   approved,
```

Comment:
Could you please remove the deprecated "approved" enum value?

Reply:
The reason I have left the approved value is that I was not sure it was not required for other comments-related operations. Ex: update, create.

But if you want me to remove it, I can do that.

Reply:
I think it is unlikely that we have two different statuses for same option on wp update and create endpoint when list api is different.

Plwase go-ahead on the removal.